CMakeLists.txt

```diff
@@ -71,10 +71,10 @@ set_package_properties(OpenCV PROPERTIES TYPE OPTIONAL)
 
 if(OpenCV_FOUND)
     set(OpenCV_dep OpenCV)
-    find_package(YARP COMPONENTS OS sig math dev cv REQUIRED)
+    find_package(YARP COMPONENTS os sig math dev cv REQUIRED)
```

Comment:
this change has nothing to do with the IMU calib dumper and causes backwards compatibility issues. Can you please revert it.

Reply:
We have been receiving the following message for a while:
"CMake Deprecation Warning at /home/leandro/libraries/install/lib/cmake/YARP/YARPConfig.cmake:146 (message):
  The 'OS' component is deprecated.  Use 'os' instead, and replace
  YARP::YARP_OS with YARP::YARP_os"

I reverted this change since it has nothing to do with the PR. But we might want to change this anytime soon, and update whatever is being troubled.